coders/exr.c

```diff
@@ -453,6 +453,10 @@ static MagickBooleanType WriteEXRImage(const ImageInfo *image_info,Image *image,
     compression=IMF_PIZ_COMPRESSION;
   if (write_info->compression == Pxr24Compression)
     compression=IMF_PXR24_COMPRESSION;
+  if (write_info->compression == DwaaCompression)
```

Comment:
Because these values are not yet available you cannot just use them here. For these two values you will need to check if the compression value is defined:

```c
#if defined(IMF_DWAA_COMPRESSION)
  if (write_info->compression == DwaaCompression)
    compression=IMF_DWAA_COMPRESSION;
#endif
```

Could you add these two new values below `IMF_B44A_COMPRESSION`? You might notice that the check is different but that is because I just pushed a patch to fix this. And could you also add support for reading these two new values in `ReadEXRImage`? I just pushed some another patch to add support for reading the compression.

Reply:
Thanks for describing in details what I needed to update! I've made an update to the merge request considering all your comments.